### PR TITLE
Update ouroboros network dependency

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

**Comment by newhoggy** [comment]
This PR is built on top of https://github.com/input-output-hk/cardano-node/pull/2656#pullrequestreview-646014960

**Review by erikd — APPROVED** [review]

File: cardano-node/src/Cardano/Node/Configuration/POM.hs [hunk 1]
```diff
@@ -256,7 +256,7 @@ defaultPartialNodeConfiguration =
     , pncLoggingSwitch = Last $ Just True
     , pncSocketPath = mempty
     , pncDiffusionMode = Last $ Just InitiatorAndResponderDiffusionMode
-    , pncSnapshotInterval = mempty
+    , pncSnapshotInterval = Last $ Just DefaultSnapshotInterval
```
Comment (Jimbo4350) [inline cardano-node/src/Cardano/Node/Configuration/POM.hs:259]: :+1:

**Review by Jimbo4350 — APPROVED** [review]
LGTM! Can we squash the dep update commits?

**Comment by newhoggy** [comment]
> LGTM! Can we squash the dep update commits?

Squashed!

**Review by nc6 — APPROVED** [review]

**Comment by newhoggy** [comment]
bors merge

**Comment by newhoggy** [comment]
bors merge

**Comment by newhoggy** [comment]
bors merge

**Comment by craigem** [comment]
bors retry

**Pull request merged**